sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -0,0 +1,43 @@
+-- test for intervals
```

Comment:
now we have a dedicated sql test file for interval, maybe we should put all interval related tests here. We can do it in followup